src/test/java/org/folio/innreach/controller/d2ir/InnReachCirculationControllerTest.java

```diff
@@ -231,4 +233,56 @@ void processCancelItemHoldRequest_whenItemIsCheckedOut() {
     assertNotEquals(BORROWING_SITE_CANCEL, transactionUpdated.getState());
   }
 
+  @Test
+  @Sql(scripts = {
+    "classpath:db/central-server/pre-populate-central-server.sql",
+    "classpath:db/inn-reach-transaction/pre-populate-inn-reach-transaction.sql"
+  })
+  void processItemReceivedRequest_whenItemIsShipped() {
+    var transaction = repository.findByTrackingIdAndCentralServerCode(PRE_POPULATED_TRACKING_ID,
+      PRE_POPULATED_CENTRAL_CODE).get();
+    transaction.setState(ITEM_SHIPPED);
+    repository.save(transaction);
+
+    var transactionHoldDTO = createTransactionHoldDTO();
+
+    var responseEntity = testRestTemplate.exchange(
+      "/inn-reach/d2ir/circ/itemreceived/{trackingId}/{centralCode}", HttpMethod.PUT,
```

Comment:
please add a constant for the endpoint path and reuse it in other tests

Reply:
Do you mean add constant for the whole path or for the operation name only?